<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFormExceptionClass</class>
 <widget class="QDialog" name="DialogFormExceptionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogFormException</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exception Name (*)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exception_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Always Pay</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Paid hours</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Paid coefficient [0, 3]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="paid_coefficient">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="paid_hours">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Extra Working Hour Paid</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Work coefficient [0, 3]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="work_coefficient">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="is_calculate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Include working hours in the overtime calculations</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/close.png</normaloff>../icon/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/check.png</normaloff>../icon/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
